packages/aws-cdk/lib/version.ts

```diff
@@ -0,0 +1,98 @@
+import { exec as _exec } from 'child_process';
+import colors = require('colors/safe');
+import { open as _open, stat as _stat } from 'fs';
+import semver = require('semver');
+import { promisify } from 'util';
+import { debug, print, warning } from '../lib/logging';
+
+const ONE_DAY_IN_SECONDS = 1 * 24 * 60 * 60;
+
+const exec = promisify(_exec);
+const open = promisify(_open);
+const stat = promisify(_stat);
+
+export const DISPLAY_VERSION = `${versionNumber()} (build ${commit()})`;
+
+function versionNumber(): string {
+  return require('../package.json').version.replace(/\+[0-9a-f]+$/, '');
+}
+
+function commit(): string {
+  return require('../build-info.json').commit;
+}
+
+export class CacheFile {
```

Comment:
It doesn't seem to be used as a "cache" file, but more as a timestamp file.

Reply:
Fixed naming.